workspace/Makefile

```diff
@@ -0,0 +1,69 @@
+.PHONY: help build start stop restart rebuild logs logs-homarr logs-redis status health shell clean clean-all
```

Comment:
Why do we need such scripts?
They have nothing to do with the actual memory change.
And they are in platform agnostic languages. Why not use an independent one, such as Python for this?